components/dialog/RotateCertificatesDialog.vue

```diff
@@ -27,7 +27,10 @@ export default {
 
   data() {
     return {
-      selectedService: '', allServices: true, errors: []
+      selectedService:   '',
+      rotateAllServices:     true,
+      errors:            [],
+      isRke2Cluster:     !!this.resources?.[0].spec?.rkeConfig
```

Comment:
provisioning clusters have a `isRke2` property. It does exactly the same thing but it's neater to use it

Reply:
I replaced it with `this.cluster.isRke2`